logins-api/android/library/src/main/java/org/mozilla/sync15/logins/MemoryLoginsStorage.kt

```diff
@@ -149,6 +194,27 @@ class MemoryLoginsStorage(private var list: List<ServerPassword>) : Closeable, L
         }
     }
 
+    private fun checkValid(login: ServerPassword) {
+        if (login.hostname == "") {
+            throw InvalidRecordException("Invalid login: Hostname is empty")
+        }
+        if (login.password == "") {
+            throw InvalidRecordException("Invalid login: Password is empty")
+        }
+        if (login.password == "") {
```

Comment:
Copy-pasta error?  Duplicated.

Reply:
Maybe you wanted to null check?

Reply:
Accidentally copypasted it. Password can't be null.